From ba0aa6690b10f12c04d2342b3040210ffa8e97cc Mon Sep 17 00:00:00 2001
From: Hans Leidekker <hans@codeweavers.com>
Date: Mon, 16 Mar 2026 13:26:35 +0100
Subject: [PATCH 1/4] include: Update definition of CERT_CHAIN_ENGINE_CONFIG.

And fix spelling of hExclusiveTrustedPeople.

Wine-Bug: https://bugs.winehq.org/show_bug.cgi?id=59531
(cherry picked from commit 2012949a0de0b550d221c5514f5632efcd8c3df2)
---
 include/wincrypt.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/wincrypt.h b/include/wincrypt.h
index 42ee4672bd9..1e0363611f8 100644
--- a/include/wincrypt.h
+++ b/include/wincrypt.h
@@ -3499,7 +3499,8 @@ typedef struct _CERT_CHAIN_ENGINE_CONFIG
     DWORD       MaximumCachedCertificates;
     DWORD       CycleDetectionModulus;
     HCERTSTORE  hExclusiveRoot;
-    HCERTSTORE  hExclusiveRootTrustedPeople;
+    HCERTSTORE  hExclusiveTrustedPeople;
+    DWORD       dwExclusiveFlags;
 } CERT_CHAIN_ENGINE_CONFIG, *PCERT_CHAIN_ENGINE_CONFIG;
 
 /* message-related definitions */
-- 
2.55.0

